fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/event/CoordinatorEventManager.java

```diff
@@ -121,6 +126,33 @@ public void clearAndPut(CoordinatorEvent event) {
                 });
     }
 
+    private Histogram getOrCreateEventProcessTimeHistogram(
+            Class<? extends CoordinatorEvent> eventType) {
+        return eventProcessTimeByType.computeIfAbsent(
+                eventType,
+                type -> {
+                    String eventTypeName = type.getSimpleName();
+                    MetricGroup eventTypeGroup =
+                            coordinatorMetricGroup.addGroup("event_type", eventTypeName);
```

Comment:
Instead of using `addGroup`, I'm thinking that could we use `variables` here? Then we can use variables to filter or combine metrics for different event types.

Reply:
I mean we can create different `MetricGroup` instances with event type in the variable maps. Such as a `class CoordinatorEventMetricGroup(EventType)` with event type as constructor parameter, and the parameter will be used in the variable maps. So that all the event reports the same metric name, but with different variables (labels). 

IIUC, the current implementation can't aggregate different event types in metric system, as the are different metric names.

Reply:
It seems you have refactored in the way I mentioned above 👍